simpleflow/storage.py

```diff
@@ -16,13 +16,26 @@ def get_connection(host):
     return connection.S3Connection(host=host)
 
 
+def sanitize_bucket_and_host(bucket):
+    """
+    if bucket is in following format : 'xxx.amazonaws.com/bucket_name',
+    Returns a 2-values tuple ('bucket_name', 'xxx.amazonaws.com')
+    """
+    if "/" in bucket:
+        host, bucket = bucket.split('/')
```

Comment:
Yes for raising a `ValueError` here if we have multiple slashes, else we may pass a wrong value at some point and get unexpected results.